examples/resources/awscc_bedrock_data_source/bedrock_data_source.tf

```diff
@@ -0,0 +1,31 @@
+resource "awscc_bedrock_data_source" "example" {
+  name              = "s3-kb"
+  knowledge_base_id = var.knowledge_base_id
```

Comment:
I think it better to show example where this point to a `awscc_bedrock_knowledge_base` resource.

The actual resource declaration itself can be omitted for brevity.

Reply:
Whatever commit I cherrypicked closed the PR and I am not able to re-open this. 
https://github.com/hashicorp/terraform-provider-awscc/pull/1699 for the updated example.